openslides/users/views.py

```diff
@@ -114,6 +119,45 @@ def reset_password(self, request, pk=None):
         else:
             raise ValidationError({'detail': 'Password has to be a string.'})
 
+    @list_route(methods=['post'])
+    @transaction.atomic
+    def mass_import(self, request):
+        """
+        API endpoint to create multiple users at once.
+
+        Example: {"users": [{"first_name": "Max"}, {"first_name": "Maxi"}]}
+        """
+        users = request.data.get('users')
+        if not isinstance(users, list):
+            raise ValidationError({'detail': 'Users has to be a list.'})
+
+        created_users = []
+        # List of all track ids of all imported users. The track ids are just used in the client.
+        imported_track_ids = []
+
+        for user in users:
+            try:
+                serializer = self.get_serializer(data=user)
+                serializer.is_valid(raise_exception=True)
+                data = serializer.prepare_password(serializer.data)
+                groups = data['groups_id']
+                del data['groups_id']
+
+                db_user = User(**data)
+                db_user.save(skip_autoupdate=True)
+                db_user.groups.add(*groups)
+                created_users.append(db_user)
+                if 'importTrackId' in user:
+                    imported_track_ids.append(user['importTrackId'])
+            except ValidationError:
```

Comment:
Please catch this exception only at the line that may raise it (serializer.is_valid(raise_exception=True) I think).